frontend/src/components/graphs/HBarChart.js

```diff
@@ -56,15 +63,18 @@ function HBarChart(props) {
   };
 
   return (
-    <div id="chart">
-      <ReactApexChart
-        options={options}
-        series={series}
-        type="bar"
-        height={400}
-        width="100%"
-      />
-    </div>
+    <>
+      <div id="chart">
+        <ReactApexChart
+          options={options}
+          series={series}
+          type="bar"
+          height={400}
+          width="100%"
+        />
+      </div>
+      <CopyGraphButton key={animationEnded} chartId='bar-chart' animationEnded={animationEnded}/>
```

Comment:
De copy to clipboard werkt niet op mobile misschien hier `{!isMobile && <>}` van maken?